air-quality-ui/system_tests/ui/city_page/city_page.spec.ts

```diff
@@ -519,3 +519,196 @@ test.describe('Charts are fully visible in 1920x1080 viewport', () => {
     })
   })
 })
+
+test.describe('High value measurements', () => {
```

Comment:
Looks good to me. Only thought is to have a more specific title like "High value measurements are displayed correctly" or similar 🤔

Reply:
done